Corona-Warn-App/src/main/res/navigation/nav_graph.xml

```diff
@@ -486,4 +492,15 @@
         android:id="@+id/debuglogFragment"
         android:name="de.rki.coronawarnapp.bugreporting.debuglog.ui.DebugLogFragment"
         android:label="DebuglogFragment" />
+
+    <!-- New Release -->
+    <fragment
+        android:id="@+id/newReleaseInfoFragment"
+        android:name="de.rki.coronawarnapp.ui.release.NewReleaseInfoFragment"
+        android:label="NewReleaseInfoFragment" >
+        <action
+            android:id="@+id/action_newReleaseInfoFragment_pop"
+            app:popUpTo="@id/newReleaseInfoFragment"
+            app:popUpToInclusive="true" />
```

Comment:
@chiljamgossow  Is this the right way? Could we just leave it out, as we `popBackStack()`?

Reply:
Still not sure about what should I do with this. I haven't changed it yet. It would work with popBackStack() also but since this is included in the navgraph features doesn't it make more sense to use it instead of pop backStack()?

Reply:
If popBackStack does the trick, use it.

Reply:
This action is dependent on the stack and the order of items, which may result in unwanted behavior. What you did is correct, however it is more complicated and therefore more prone to bugs.